### PR TITLE
Update PrimeNumber.java

### PR DESCRIPTION
**Pull request by akankshaJain17**
To get reviewed

### CODE REVIEW

**Comment by akankshaJain17** [comment]
Can you review

**Comment by akankshaJain17** [comment]
@brainydexter can you please review ?

File: PrimeNumber.java [hunk 1]
```diff
@@ -12,6 +12,8 @@ public static void main (String[] args)
 		boolean c = isPrime(n);
 		System.out.println(n + "is a Prime Number :" + c );
 	 
+	 
+	 
 	}
 	
 	public static boolean isPrime(int n)
```
Comment (brainydexter) [inline PrimeNumber.java:19]: Why don't you add runtime complexity information here ? I will see if it's correct and then approve the changes!

Comment (akankshaJain17) [inline PrimeNumber.java:19]: @brainydexter - I did add it....

**Pull request merged**